WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginSiteAddressFragment.java

```diff
@@ -502,4 +504,22 @@ private Map<String, String> createConnectSiteInfoProperties(ConnectSiteInfoPaylo
         properties.put(KEY_SITE_INFO_CALCULATED_HAS_JETPACK, Boolean.toString(hasJetpack));
         return properties;
     }
+
+    private String stripKnownPaths(String url) {
+        String cleanedXmlrpcSuffix = UrlUtils.removeXmlrpcSuffix(url);
+
+        // Make sure to use a valid URL so that DiscoveryUtils#stripKnownPaths is able to strip paths
+        String scheme = Uri.parse(cleanedXmlrpcSuffix).getScheme();
+        String urlWithScheme;
+        if (scheme == null) {
+            urlWithScheme = UrlUtils.addUrlSchemeIfNeeded(cleanedXmlrpcSuffix, false);
+        } else {
+            urlWithScheme = cleanedXmlrpcSuffix;
+        }
+
+        String cleanedUrl = DiscoveryUtils.stripKnownPaths(urlWithScheme);
+
+        // Revert the scheme changes
+        return scheme == null ? UrlUtils.removeScheme(cleanedUrl) : cleanedUrl;
```

Comment:
`DiscoveryUtils.stripKnownPaths` requires a valid URL, so if we pass a URL without the scheme, the suffixes don't get deleted, to get around this, I'm doing this dance of adding the scheme then removing it (only if the initial URL didn't have it).